openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -160,7 +164,7 @@ const Carousel = {
 
                 if (!loadMore.locked && !loadMore.allDone && isOn2ndLastPage) {
                     loadMore.locked = true; // lock for critical section
-                    slick.addSlide('<div class="carousel__item carousel__loading-end">Loading...</div>');
+                    slick.addSlide('<object type="image/svg+xml" data="/static/images/spinner.svg"></object>');
```

Comment:
```suggestion
                    slick.addSlide('<div class="carousel__item carousel__loading-end">i18nValues['loading']</div>');
```
Use localized "loading" string here.